rabbitmq_status.py

```diff
@@ -93,8 +93,15 @@ def main():
 
     if r.ok:
         resp_json = r.json()  # Parse the JSON once
-        if any('channels' in connection and connection['channels'] > 1 for connection in resp_json):
-            status_err('Detected RabbitMQ connections with multiple channels. Please check RabbitMQ and all Openstack consumers')
+        # We routinely see connection['channels'] == 2, so we cannot alarm if
+        # connection['channels'] > 1.  Note that connection['channels'] does
+        # not appear to be the total number of channels using the connection,
+        # but rather the channel number which identifies it within a
+        # connection.  We'll assume if we have a channel w/ number 11 or
```

Comment:
I'm a bit confused here, https://github.com/rcbops/rpc-maas/commit/fe61c70666693f14eb9fcc92431c49f278b60068, was made because the check was being done against the channel number. It now sounds like connection['channels'] does the same thing as channel['number'] in the previous commit which seems strange.

Assuming that connection['channels'] is the channel number what is the significance of the number being > 10?